tests/scripts/kubeadm-install.sh

```diff
@@ -47,4 +47,9 @@ sudo apt-get update
 wait_for_dpkg_unlock
 sleep 5
 wait_for_dpkg_unlock
+
+# We install the specific version of kubernetes-cni for aws_1.11.x test.
+# kubelet in this test requires this version. If newer packages are necessary
+# for the other testk, these ones can be installed online.
```

Comment:
Which other tests are you referring to?
```suggestion
# for the other tests, these can be installed online.
```

Reply:
"Other tests" means aws_1.1[3-8].x test here. "The tests for other Kubernetes versions" or something is more preferable expression?

Reply:
Does the following comment make sense?

```
# We install the specific version of kubernetes-cni for aws_1.11.x test.
# kubelet in this test requires the version. If the newer versions are
# necessary in the tests of other Kubernetes versions, kubernetes-cni is
# updated in the kubelet installation.
```

Reply:
Yes, that is clear thank you